### DISCUSSION TITLE
Can we get an ETA for Django 4.2

### DISCUSSION THREAD

**Discussion by imolitor** [cookiecutter/cookiecutter-django#4381] [General]
Django 4.2 is an LTS release, hence it would be great to have Cookiecutter Django cover version 4.2 imho

**Comment by foarsitter** [comment]
You can track the progress in this issue: https://github.com/cookiecutter/cookiecutter-django/issues/4252 

Upgrading to 4.2 will work I suppose but not al libs support have support for 4.2 formally.

**Comment by browniebroke** [comment]
Have you tried to upgrade a project generated with cookiecutter to Django 4.2? Does it work? If it doesn't work, what doesn't work? It's usually down to 3rd party Django packages, if a specific one doesn't work, the best way for you to help us is to contribute to that project.

**Comment by luzfcb** [comment]
Fixed on https://github.com/cookiecutter/cookiecutter-django/pull/4393, and the first version is included on the tag https://github.com/cookiecutter/cookiecutter-django/releases/tag/2023.06.28
